### ISSUE TITLE
passing null for name to add function causes TypeError

### ISSUE THREAD

**Issue by theycallmecoach** [svg-sprite/svg-sprite#816]
```
/app/node_modules/svg-sprite/lib/svg-sprite.js:93
             name = trimStart(name.trim(), `${path.sep}.`) || path.basename(file);
                                   ^
 
 TypeError: Cannot read properties of null (reading 'trim')
```
 
Found when updating from older 2.0.2 version.  My code was incorrectly passing null but it didn't crash ... latest version crash.  I've since fixed my code but thought i should let you know the null check needs adjusting...

**Comment by XhmikosR** [comment]
Thank for the issue. This is the same as https://github.com/svg-sprite/svg-sprite/issues/598#issuecomment-1536036933

@strager we are better off refactoring the code to check for null too and maybe drop the default params if they are not needed.

**Comment by strager** [comment]
Proposed fix in #817.

**Comment by Kreeg** [comment]
Fixed in https://github.com/svg-sprite/svg-sprite/pull/819
